### PR TITLE
Quick Start V2: UI fix

### PR DESCRIPTION
**Pull request by danielebogo**
Refs. #10860 

This PR adds some UI fix:
### Quick Start Section
- Remove disclosure indicator on iPhone
- Fix separator inset width

screenshot 2019-02-08 at 08 58 25

### Checklist
- iPhone: separators are edge to edge based on the safe area layout guide
- iPad: separators are following the readable layout guide
- iPad: the list has a section header and the first todo row has a top separator
- The icon imageview is aligned correctly

**iPhone portrait**
simulator screen shot - iphone x - 2019-02-08 at 08 53 26

**iPhone landscape**
simulator screen shot - iphone x - 2019-02-08 at 08 53 21

**iPad**
img_0133

**To Test:**
* Put `return true` in `shouldShowQuickStartChecklist()`
* Open the blog detail on iPhone to check the QS section rows
* Open the checklist to check the separator insets (on both iPhone and iPad)

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift [hunk 1]
```diff
@@ -53,15 +67,38 @@ class QuickStartChecklistCell: UITableViewCell {
...
-            WPStyleGuide.configureLabel(titleLabel, textStyle: .headline)
-            WPStyleGuide.configureLabel(descriptionLabel, textStyle: .subheadline)
+            setupConstraints()
         }
     }
 
     static let reuseIdentifier = "QuickStartChecklistCell"
 }
+
+extension QuickStartChecklistCell {
```
Comment (nheagy) [inline WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift]: I think this should be a `private extension`?

**Comment by loremattei** [comment]
I'm moving this to 11.9 since 11.8 has been cut. If you want it in 11.8, please feel free to target the release branch and to ping me to build a new beta.

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift (hunk 1 above)
Comment (danielebogo) [reply WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift]: Yes you know that I like that solution but then I saw this: https://google.github.io/swift/#access-levels
What do you think?

Comment (nheagy) [reply WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift]: I definitely think this is something that's inconsistent in our code base. I'm fine with you picking whichever you prefer for this PR! There's a `private extension` added in `QuickStartListTitleCell.swift`, so just ensure these two are consistent?

**Comment by SylvesterWilmott** [comment]
Looks great @danielebogo !

**Comment by danielebogo** [comment]
Thanks @nheagy @SylvesterWilmott

**Pull request merged**